### PR TITLE
[stable12] Don't send emails to disabled users

### PR DESCRIPTION
**Pull request by nickvergessen**

### CODE REVIEW

File: lib/MailQueueHandler.php [hunk 1]
```diff
@@ -163,10 +164,16 @@ public function sendEmails($limit, $sendTime, $forceSending = false, $restrictEm
 		$deleteItemsForUsers = [];
 		$this->activityManager->setRequirePNG(true);
 		foreach ($affectedUsers as $user) {
+			if (isset($userEnabled[$user]) && $userEnabled[$user] === 'no') {
```
Comment (MorrisJobke) [inline lib/MailQueueHandler.php]: For me this was set to `"false"`:

bildschirmfoto 2017-09-28 um 17 49 38

Comment (MorrisJobke) [reply lib/MailQueueHandler.php]: 🙌 to type safety ...

Comment (nickvergessen) [reply lib/MailQueueHandler.php]: argl

Comment (nickvergessen) [reply lib/MailQueueHandler.php]: Fixed

**Comment by nickvergessen** [comment]
Master follow up https://github.com/nextcloud/activity/pull/208

**Pull request merged**